packages/ui/src/working-groups/types/WorkingGroupOpening.ts

```diff
@@ -28,6 +35,12 @@ export interface WorkingGroupOpening {
     member: Member
     status: string
   }[]
+  createdAtBlock: {
+    id: string
+    number: number
+    network: Network
+    createdAt: string
+  }
```

Comment:
Could you use `Block` from `@/common/types`? It looks like it also needs an update to have a `timestamp` field.